sabnzbd/utils/certgen.py

```diff
@@ -57,17 +58,17 @@ def generate_local_cert(private_key, days_valid=3560, output_file="cert.cert", L
 
     # Then the host IP addresses, add with x509.IPAddress()
     # Inside a try-except, just to be sure
-    try:
-        import ipaddress
+    import ipaddress
 
+    try:
         san_list.append(x509.IPAddress(ipaddress.IPv4Address("127.0.0.1")))
         san_list.append(x509.IPAddress(ipaddress.IPv6Address("::1")))
 
         # append local v4 ip
         mylocalipv4 = localipv4()
         if mylocalipv4:
             san_list.append(x509.IPAddress(ipaddress.IPv4Address(str(mylocalipv4))))
-    except:
+    except (crypto_exceptions.InternalError, ipaddress.AddressValueError, ipaddress.NetmaskValueError):
```

Comment:
Are you sure these are all that can occur?

Reply:
These were the ones that popped up when I ran a simple test, but you're absolutely right: there are likely more that can occur.  What I'll try and do is add some broader exceptions, but not leaving a bare exception, like BufferError or LookupErrors, that capture more without interfering with real bugs.